### PR TITLE
APIVIew - Make GitHub token optional

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Github token is only required when testing feature related to API reviews created from PR. But this field is currently mandatory since it is used to check if cleanup is required. Change is to skip cleanup eligibility check when github token is not set.

### CODE REVIEW

**Comment by praveenkuttappan** [comment]
@JonathanGiles FYI. This is to make github token optional.

**Comment by JonathanGiles** [comment]
Thanks!

File: eng/scripts/Create-Apiview-Token-Python.ps1 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[CmdletBinding()]
```
Comment (weshaggard) [inline eng/scripts/Create-Apiview-Token-Python.ps1:1]: Looks like you push this change via the wrong branch which contains other changes as well.

Comment (praveenkuttappan) [reply eng/scripts/Create-Apiview-Token-Python.ps1:1]: I just realized it after I pushed one more change. Sorry about that.

**Review by weshaggard — APPROVED** [review]

**Review by JoshLove-msft — APPROVED** [review]

**Comment by praveenkuttappan** [comment]
Fixes #3783

**Pull request merged**